src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -62,6 +62,10 @@
                 "title": "Max Static Code Analysis Penalty",
                 "description": "You can choose a value between 0 and 100 percent or leave the field empty. If the maximum penalty is 20% for a 10 point exercise, a maximum of two points can be deducted because of issues found by static code analysis. If the field is left empty, no penalty limit will be enforced."
             },
+            "recreateBuildPlans": {
+                "title": "Recreate Build Plans",
+                "description": "Activate this options to create new build plans and don't copy them from the imported exercise."
```

Comment:
```suggestion
                "description": "Activate this option to create new build plans instead of copying them from the imported exercise."
```